rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -522,6 +522,35 @@ def test_exec(config, ssh_con):
                                     raise TestExecError(
                                         "Bucket stats for non-existent bucket should return failure (2) or ENOENT."
                                     )
+                    if config.test_bi_purge is True:
```

Comment:
We can replcae this line with `if config.test_bi_purge:`

Reply:
Definitely, yes.